core/table.ts

```diff
@@ -664,7 +671,8 @@ export class Table {
       uniqueKeys.forEach((uniqueKey, index) => {
         const uniqueKeyAssertion = this.session.assert(
           `${this.proto.target.schema}_${this.proto.target.name}_assertions_uniqueKey_${index}`,
-          ctx => this.session.adapter().indexAssertion(ctx.ref(this.proto.target), uniqueKey)
+          ctx => this.session.adapter().indexAssertion(ctx.ref(this.proto.target), uniqueKey),
+          this.proto.tags
```

Comment:
this only works for when the caller calls `assertions(...)` *after* calling `tags(...)`.

consider the case where the caller adds another tag to the table *after* adding assertions. if you look at the equivalent for `disabled`, you will see that `disabled(...)` copies that through to any attached assertions - you need to do the same for tags.

Reply:
Done. I would edit a test for this case, but I don't see any pre-existing that test this order of events